Riot/ViewController/SegmentedViewController.m

```diff
@@ -17,7 +17,9 @@
 
 #import "SegmentedViewController.h"
 
-#import "AppDelegate.h"
+#import "RiotDesignValues.h"
+//#import "RageShakeManager.h"
```

Comment:
You can remove rageShakeManager here. It should be defined by the child VCs.
Please clean commented lines